components/resources/library/src/commonMain/kotlin/org/jetbrains/compose/resources/Resource.kt

```diff
@@ -14,7 +14,6 @@ annotation class InternalResourceApi
  * @property id The ID of the resource.
  * @property items The set of resource items associated with the resource.
  */
-@ExperimentalResourceApi
```

Comment:
Not for this PR, but because we are removing experimental, we have to support API backward compatibility, and better to apply https://github.com/Kotlin/binary-compatibility-validator plugin for that